fbgemm_gpu/codegen/training/forward/embedding_forward_split_kernel_template.cu

```diff
@@ -155,6 +153,102 @@ using namespace fbgemm_gpu;
     {%- endif %}
 {%- endmacro %}
 
+{#-/*
```

Comment:
There is one question. Should we keep the comment in generated source files?

(Just a concern, not a blocking factor)